torch/csrc/distributed/rpc/tensorpipe_utils.h

```diff
@@ -44,12 +43,12 @@ class TensorpipeDeviceTypeConverter {
   virtual ~TensorpipeDeviceTypeConverter() = default;
 };
 
-extern C10_API std::array<
```

Comment:
I had added `C10_API` myself not even a day ago but I realized now that `TORCH_API` might be more appropriate? (I don't know the difference, but since this is part of libtorch...)